example/docker-compose/docker-compose.yaml

```diff
@@ -60,21 +59,20 @@ services:
     ports:
       - "9001:9001"
 
-  # tracing load generator
-  synthetic-load-generator:
-    profiles: [agent] # Should only be run if the Agent is present
-    image: omnition/synthetic-load-generator:1.0.25
-    volumes:
-      - ./load-generator:/etc/load-generator
-    environment:
-      - TOPOLOGY_FILE=/etc/load-generator/load-generator.json
-      - JAEGER_COLLECTOR_URL=http://agent:14268
+  hotrod:
```

Comment:
Might be good to dogfood [xk6-client-tracing](https://github.com/grafana/xk6-client-tracing) like the [Tempo repo does](https://github.com/grafana/tempo/blob/main/example/docker-compose/agent/docker-compose.yaml#L5-L11).

Reply:
Ah, I didn't know about this one. Maybe we can leave this upgrade for another PR, because we've been creeping scope in this one for a bit already.